### DISCUSSION TITLE
[RxJS] 認識 Scheduler | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#65] [Announcements]
# [RxJS] 認識 Scheduler | 全端開發人員天梯

今天我們來認識一下 RxJS 的 Scheduler，雖然在一般使用 RxJS 開發應用程式時幾乎不會用到 Scheduler，但 Scheduler 可以說是控制 RxJS 至關重要的角色，偶爾也有可能會需要使用 Scheduler 來調整事件發生時機！到底什麼是 Scheduler？就讓我們繼續看下去吧！

https://fullstackladder.dev/blog/2020/10/19/mastering-rxjs-34-introduce-scheduler-of-rxjs/

**Comment by Fiv5** [comment]
感谢你的文章，解答了我作为一个新入门 RxJS 小白的疑惑 :)
